src/Pos/Wallet/Web/State/Storage.hs

```diff
@@ -81,6 +81,7 @@ type CustomAddresses = HashMap (CId Addr) (HeaderHash)
 
 data WalletInfo = WalletInfo
     { _wiMeta         :: CWalletMeta
+    , _wiId           :: Int           -- ^ number of wallet in creation order
```

Comment:
I'll take your word for it :) It's strange though, looking at the code, it seems that we are already passing the current time to `createWallet`. I would think it could just be stored as-is, or if that param has a different meaning, we can pass another one.